ansible/setup.yml

```diff
@@ -92,3 +92,7 @@
   - name: generate invoker certificates
     when: invoker.protocol == 'https'
     local_action: shell "{{ playbook_dir }}/files/genssl.sh" "{{ invoker.ssl.cn }}" "server" "{{ playbook_dir }}/roles/invoker/files" {{ invoker.ssl.keystore.password }} {{ invoker.ssl.keyPrefix }} "generateKey"
+
+  - name: generate lean controller plugin invoker certificates
```

Comment:
why is this necessary?

Reply:
Lean controller ansible setup calls invoker ansible, which requires invoker certificates

Reply:
is the invoker certificate required though or can it be predicated on `!lean` as well?

Reply:
lean predicate will require adding new variable to whisk.conf as well

if it's ok, currently resolved it by removing "generate invoker certificates" block to avoid code duplication. using fact that ansible searches for invoker certificates in ansible/files as well and not only /roles/invoker/files

Reply:
i still don't understand the need for these certs in lean mode.
but because the files move as a result of this change, you need to also update the git ignores.

Reply:
removed this change from the setup.yml.
fixed the issue by adding full path to certificates in invoker deploy.yml, similar to how kafkas certificates handled by invokers ansible.